modular_skyrat/modules/cargo/code/items/AFAD.dm

```diff
@@ -0,0 +1,151 @@
+// Copypasted just to avoid having to make non-modular edits to the medbeam.
```

Comment:
bad

Reply:
https://www.masterclass.com/articles/how-to-write-descriptive-sentences